src/goto-symex/field_sensitivity.cpp

```diff
@@ -68,10 +71,15 @@ void field_sensitivityt::apply(exprt &expr, bool write) const
       // place the entire member expression, not just the struct operand, in an
       // SSA expression
       ssa_exprt tmp = to_ssa_expr(member.struct_op());
```

Comment:
I think this could be a reference instead of a copy since expr doesn't seem to be used afterwards

Reply:
I don't think that would be a big improvement.